src/components/about/About.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export default function About(){
```

Comment:
What do you think about moving 'export default' to the bottom of the page similar to the defaull App.js structure? It's general practice to have the exporting separate from the function declaration.

Reply:
for sure!